documentation/src/main/asciidoc/userguide/Preface.adoc

```diff
@@ -44,9 +44,9 @@ While having a strong background in SQL is not required to use Hibernate, it cer
 Probably even more important is an understanding of data modeling principles.
 You might want to consider these resources as a good starting point:
 
-* http://en.wikipedia.org/wiki/Data_modeling[Data Modeling Wikipedia definition]
+* http://en.wikipedia.org/wiki/Data_modeling[Data modeling Wikipedia definition]
 * http://www.agiledata.org/essays/dataModeling101.html[Data Modeling 101]
 
-Understanding the basics of transactions and design patterns such as _Unit of Work_ <<Bibliography.adoc#PoEAA,PoEAA>> or _Application Transaction_ are important as well.
+Understanding the basics of transactions and design patterns such as _Unit of Work_ (<<Bibliography.adoc#PoEAA,PoEAA>>) or _Application Transaction_ are important as well.
```

Comment:
I am always curious what is the exact meaning of `Application Transaction`, btw?

Reply:
It's referring to the notion of "transaction" in the domain. Essentially, it's the concept of a transaction in the final domain, which might be different than the implementation details.

Reply:
Thanks. I appreciate your explanation. I am reading the book of `PoEAA` and found it is amazing.

Reply:
right, the book will have a much better explanation :+1: